package.json

```diff
@@ -8,7 +8,7 @@
     "Nicholas Hwang <nhwang@hubspot.com>"
   ],
   "engines": {
-    "node": ">=6.9"
+    "node": ">=6.2.2"
```

Comment:
Could we just drop this all the way to `>=6` rather than pinning it to exactly `6.2.2`?

Also, I believe these configurations _should_ work on the lower common version of the dependencies e.g. `eslint` so it's possible this value is even lower. This engine version was primarily set to ensure that the version of `node` can support `async/await` in the specs, but shouldn't affect actual usage of the configurations.